File: Mga Codes/ArmstrongNumber.c
#include <stdio.h>

int main ()
{
	int number, sum=0, store, mod;
	printf(" Is it an Armstrong Number or Not");
	printf("\n\nEnter number: ");
	scanf("%d", &number);
	store=number;
	while(number>0){
		mod=number%10;
		sum=sum+(mod*mod*mod);
		number=number/10;
	}
if(store==sum) printf("\n%d is an Armstrong Number",store );
else printf("Not an Armstrong Number");
}

File: beginner-projects-solutions/ArmstrongNumber.py
number = int (input ("Enter a number: "))
tempNumber = number
sum = 0
while (number > 0):
    mod = int(number % 10)
    sum = int(sum + (mod*mod*mod))
    number = int(number / 10)
print (sum)
if (tempNumber == sum):
    print(f"{tempNumber} is an Armstrong Number")
else : 
    print(f"{tempNumber} is not an Armstrong Number")
